src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -191,53 +191,42 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
         const preferredSearchToken = new CancellationTokenSource();
         this.findPreferredInProgress.set(document, preferredSearchToken);
 
-        if (!this.isLocalLaunch) {
```

Comment:
Please ignore this code, blocked by https://github.com/microsoft/vscode-jupyter/pull/6342
If that PR isn't merged, I'll revert the changes.
The changes in this file are in https://github.com/microsoft/vscode-jupyter/pull/6342